recipes/update.rb

```diff
@@ -22,63 +22,12 @@
 
 include_recipe 'wsus-client::configure'
 
-# API documentation: http://msdn.microsoft.com/en-us/library/windows/desktop/aa387099.aspx
-powershell_script 'wsus_update_script' do # ~FC009
-  code <<-EOH
-    $session = New-Object -com 'Microsoft.Update.Session'
-
-    Write-Host 'Looking for updates...'
-    $searcher = $session.CreateUpdateSearcher()
-    $foundUpdates = $searcher.Search('IsInstalled=0').Updates
-
-    Write-Host 'Update(s) to download: ' $foundUpdates.Count
-    if ($foundUpdates.Count -eq 0) {
-      Write-Warning 'No update available.'
-      exit 0
-    }
-
-    $downloader = $session.CreateUpdateDownloader()
-    $downloader.Updates = $foundUpdates
-
-    Write-Host 'Downloading updates...'
-    $downloadResult = $downloader.Download()
-    # ResultCode values: http://msdn.microsoft.com/en-us/library/windows/desktop/aa387095.aspx
-    if (($downloadResult.HResult -ne 0) -or (($downloadResult.ResultCode -ne 2) -and ($downloadResult.ResultCode -ne 3))) {
-      Write-Host 'Download failed.' -f red
-      exit 1
-    }
-
-    Write-Host 'Preparing updates...'
-    $updatesToInstall = New-object -com 'Microsoft.Update.UpdateColl'
-    foreach ($update in $foundUpdates) {
-      if (! $update.isdownloaded) {
-        Write-Warning "Update $($update.Title) not downloaded."
-        continue
-      }
-      if (! $update.EulaAccepted) {
-        Write-Host "Accepting EULA for $($update.Title)." -F green
-        $update.AcceptEula()
-      }
-      $updatesToInstall.Add($update) | out-null
-    }
-
-    $installer = $session.CreateUpdateInstaller()
-    $installer.ForceQuiet = $true
-    $installer.Updates = $updatesToInstall
-
-    $installationResult = $installer.Install()
-    if (($installationResult.HResult -ne 0) -or ($installationResult.ResultCode -ne 2)) {
-      Write-Host "Installation failed. (Error code $($installationResult.HResult))" -f red
-      exit 1
-    }
+if node['wsus_client']['download_only']
+  actions_to_perform = [:download]
+else
+  actions_to_perform = [:download, :install]
+end
```

Comment:
Yes it could be possible, but I don't think we need it; it'ld be easy to add the feature when needed, don't you think ?